web/src/components/BasePage/BasePage.jsx

```diff
@@ -1,17 +1,43 @@
 import React from 'react';
+import { connect } from 'react-redux';
 
 import './BasePage.css';
 
-const BasePage = ({ bases, selectedBase }) => {
+function BasePage({ selectedBase }) {
+  if (selectedBase) {
+    return (
+      <div className='BasePage'>
+        <p className='BasePage-name'>{selectedBase.name}</p>
+        <p className='BasePage-score'>
+          <span class='fa fa-star yellow'></span> {selectedBase.score}
+        </p>
+        <img src={selectedBase.icon} alt={selectedBase.name} />
+      </div>
+    );
+  }
   return (
     <div className='BasePage'>
-      <p className='BasePage-name'>{bases[selectedBase].name}</p>
-      <p className='BasePage-score'>
-        <span class='fa fa-star yellow'></span> {bases[selectedBase].score}
-      </p>
-      <img src={bases[selectedBase].icon} alt={bases[selectedBase].name} />
+      <p className='BasePage-name'>Welcome to TouchBase</p>
     </div>
   );
+}
+
+// const BasePage = ({ bases, selectedBase }) => {
```

Comment:
are you keeping this for something? just remove unneccessary comments, you can always revert later